R/handler_warnings.R

```diff
@@ -0,0 +1,30 @@
+#' @title Capture Warnings
+#'
+#' @description This function captures the warnings generated by an
+#' expression without printing them to the console. It returns both the
+#' result of the expression and the captured warnings. It is useful for
+#' avoiding warnings appearing in the middle of a routine (for instance,
+#' an iterative routine).
+#'
+#' @param expr An expression to evaluate.
+#' @param warn_env An optional environment to store the warnings.
```

Comment:
Specify that the function does not affect the user's environment and explain the usage of `invokeRestart("muffleWarning")`